reactNativeProject/src/screens/ListOfBooks/index.js

```diff
@@ -6,15 +6,21 @@ import books from "../../constants/books";
 
 const extractKey = ({ id }) => id;
 
-function ListOfBooks() {
+function ListOfBooks({ navigation }) {
   return (
     <ScrollView>
       <Text style={styles.container}>Screen One</Text>
       <FlatList
         styles={styles.list}
         data={books}
         renderItem={({ item }) => (
-          <Item img={item.image_url} title={item.title} author={item.author} />
+          <Item
+            img={item.image_url}
+            title={item.title}
+            author={item.author}
+            genre={item.genre}
+            navigation={navigation}
```

Comment:
Don't pass navigation to the Item. Use [withNavigation](https://reactnavigation.org/docs/en/with-navigation.html)